examples/52_tanstack-router/README.md

```diff
@@ -0,0 +1,6 @@
+# Example
+
+To run this example:
+
+- `npm install` or `yarn`
+- `npm start` or `yarn start`
```

Comment:
We don't need README for now. Let's reconsider it in the future.

Reply:
removed